plugins/modules/vmware_host_facts.py

```diff
@@ -26,6 +20,9 @@
       module will throw an error.
     - VSAN facts added in 2.7 version.
     - SYSTEM fact uuid added in 2.10 version.
+    - Powerstate fact added in VMware collection 1.0.0.
```

Comment:
I'm guessing this will need to be changed to 2.0.0?

Reply:
Since we've released 2.5.0 a few days ago, it should be 2.6.0 ;-)

However, I doubt that this PR fixes the original issue (see the comments from @anxstj and @sky-joker above). I think I should have a closer look into this and work on a new PR.

I won't close this one yet, though, because I'm afraid I will forget to do this once it's closed.